index.html

```diff
@@ -14,10 +14,16 @@ <h3>About</h3>
 group provides infrastructure and services such as build systems, testing and continuous
 integration, code format guidelines, linting and static analysis, release management and software
 distribution and data persistency. This is possible due to the kind support of the EP-SFT group.
-
 </p>
 
 
+<h3>FCC Software Meetings</h3>
+
+Our regular software meetings are organized usually on the last Monday of the
+month and cover generally useful tooling. The Indico category can be found
```

Comment:
I would replace "and cover generally useful tooling." with something more general, such as
"to discuss general matters related to software and computing, including tools and resources."

Reply:
changed